tests/data_validation/great_expectations/checkpoints/past_articles_count_validations.yml

```diff
@@ -3,7 +3,7 @@ config_version: 1.0
 template_name:
 module_name: great_expectations.checkpoint
 class_name: Checkpoint
-run_name_template: 'newsletter automation-past articles validation'
+run_name_template: 'Ready Past Week Article with Title and Description'
```

Comment:
I think this checks the Past Articles count and not title and description.

Reply:
Update run_name_template: 'Past Article Count'